Assets/Scripts/Dialogue scripts/Dialogue.cs

```diff
@@ -1,197 +1,184 @@
 
-    public void Interact(DeathMovement playerInteraction)
-    {
-        if (_movement != null) _movement.enabled = false;
-        if (accessItem != null)
+        public void Interact(DeathMovement playerInteraction)
         {
-            if (accessItem.activeSelf == false) StartCoroutine(RunParagraphCycle());
+            if (_movement != null) _movement.enabled = false;
+            if (_accessItem != null)
+            {
+                if (_accessItem.activeSelf == false) StartCoroutine(RunParagraphCycle());
 
-            else StartCoroutine(RunHiddenParagraphCycle());
-        }
+                else StartCoroutine(RunHiddenParagraphCycle());
+            }
 
-        else if (item != null)
-        {
-            if (item.activeSelf == true) StartCoroutine(RunHiddenParagraphCycle());
+            else if (_item != null)
+            {
+                if (_item.activeSelf == true) StartCoroutine(RunHiddenParagraphCycle());
+
+                else StartCoroutine(RunParagraphCycle());
+            }
 
             else StartCoroutine(RunParagraphCycle());
         }
 
-        else StartCoroutine(RunParagraphCycle());
-    }
+        public IEnumerator Play(String pageText)
+        {
+            var sb = new StringBuilder();
 
-    public IEnumerator Play(String pageText)
-    {
-        var sb = new StringBuilder();
+            var letters = pageText.ToCharArray();
 
-        var letters = pageText.ToCharArray();
+            foreach (var letter in letters)
+            {
+                sb.Append(letter);
 
-        foreach (var letter in letters)
-        {
-            sb.Append(letter);
+                _pageRender.RenderPageText(sb.ToString());
 
-            _pageRender.RenderPageText(sb.ToString());
+                yield return new WaitForSeconds(0.0375f);
+            }
 
-            yield return new WaitForSeconds(0.0375f);
+       
+            yield return null;
         }
 
-       
-        yield return null;
-    }
+        public IEnumerator RunParagraphCycle()
+        {
+            _dialogueBox.SetActive(true);
 
-    public IEnumerator RunParagraphCycle()
-    {
-        dialogueBox.SetActive(true);
+            _rayCast.SetActive(false);
 
-        rayCast.SetActive(false);
+            int paragraphCounter = 0;
 
-        int paragraphCounter = 0;
+            Coroutine currentRoutine = null;
 
-        Coroutine currentRoutine = null;
+            while (paragraphCounter < _textBox.Length)
+            {
+                if (currentRoutine != null) StopCoroutine(currentRoutine);
 
-        while (paragraphCounter < _textBox.Length)
-        {
-            if (currentRoutine != null) StopCoroutine(currentRoutine);
+                currentRoutine = StartCoroutine(Play(_textBox[paragraphCounter]));
 
-            currentRoutine = StartCoroutine(Play(_textBox[paragraphCounter]));
+                ++paragraphCounter;
 
-            ++paragraphCounter;
+                yield return new WaitForSeconds(0.0375f);
```

Comment:
Magic number exists here. I'd advise allowing this to be configurable via the inspector. This will allow you to give it a meaningful name while also making for easier tweaking.